compose.yaml

```diff
@@ -84,6 +85,8 @@ x-airflow-common:
     AIRFLOW_VAR_SUL_PUB_KEY: ${AIRFLOW_VAR_SUL_PUB_KEY}
     AIRFLOW_VAR_DEV_LIMIT: ${AIRFLOW_VAR_DEV_LIMIT}
     AIRFLOW_VAR_DATA_DIR: /opt/airflow/data
+    AIRFLOW_VAR_GOOGLE_CONNECTION: google_cloud_default
+    AIRFLOW_VAR_GOOGLE_DRIVE_ID: ${AIRFLOW_VAR_GOOGLE_DRIVE_ID}
     AIRFLOW_VAR_PUBLISH_DIR: /opt/airflow/data/latest
```

Comment:
same comments apply above from `compose.prod.yaml` file, it's all the same stuff